packages/grid_client/src/modules/calculator.ts

```diff
@@ -49,6 +49,13 @@ class Calculator {
   async calSU(options: SUModel) {
     return options.hru / 1200 + options.sru / 200;
   }
+
+  @expose
+  @validateInput
+  async calNU(options: NUModel) {
+    // what is this number
+    return (options.nu * 1000) / 1e7;
```

Comment:
```suggestion
    return options.nu / 1e4;
```

Reply:
@zaelgohary no, keep it as * 1000 / 1e7 (this explains the flow)